src/Symfony/Resources/config/services/ezplatform.yaml

```diff
@@ -38,3 +38,8 @@ services:
             $views: { field: '%ezcontentquery_field_view%', item: '%ezcontentquery_item_view%' }
         tags:
             - { name: kernel.event_subscriber }
+
+    ezpublish.fieldType.indexable.unindexed:
```

Comment:
This causes an override of existing `ezpublish.fieldType.indexable.unindexed` service, so while it will work for Query Field Type, it will cause other unindexed to fail.
Steps to reproduce:
```
php ./bin/console debug:container ezpublish.fieldType.indexable.unindexed
```
before the change and after the change

Reply:
Also, it doesn't really make sense to name it that way, does it ?

Reply:
Nice catch @alongosz :wink: